libnmstate/nm/connection.py

```diff
@@ -24,6 +24,12 @@
 from . import nmclient
 from .active_connection import ActiveConnection
 
+NM_DEVICE_DISCONNECTED = (
+    nmclient.NM.ActiveConnectionStateReason.DEVICE_DISCONNECTED
```

Comment:
Please do not do this at the global scope. Please do it in the method that uses it, the cost is not high.
Doing it here may mess up tests and it becomes another point when we may fail if libnm or NM are not available.